deployment/src/strongmind_deployment/container.py

```diff
@@ -302,6 +303,12 @@ def __init__(self, name, opts=None, **kwargs):
 
         if self.kwargs.get('autoscaling'):
             self.autoscaling()
```

Comment:
Does it make sense to refactor this out into a "WebAutoscaleComponent" based on the pattern in this PR?

Reply:
We talked about it but wanted to start with smaller changes first as working with the tests are already complicating the changes.